spring-cloud-gcp-core/pom.xml

```diff
@@ -18,5 +18,9 @@
 			<groupId>com.google.cloud</groupId>
 			<artifactId>google-cloud-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-core</artifactId>
```

Comment:
Why do we need this explicitly?

I don't see any commit comment explaining the change.

Thanks

Reply:
Apologies, this is needed because the core module now uses `Resource`.

Reply:
Oh! I see now. Sorry. I thought that was a root pom 😄